plugin/forward/health.go

```diff
@@ -40,7 +44,11 @@ func (h *dnsHc) SetTLSConfig(cfg *tls.Config) {
 	h.c.TLSConfig = cfg
 }
 
-// For HC we send to . IN NS +norec message to the upstream. Dial timeouts and empty
+func (h *dnsHc) SetRecursionDesired(recursionDesired bool) {
```

Comment:
probably not needed.

Reply:
Removed

Reply:
either github is not working, or you have pushed your code?

Reply:
Sorry, I thought i removed this function, but it was a similar one from another file. I'm not sure how else to bring the boolean from the setup-function over to the dnsHc without changing a couple of function-signatures.

Reply:
yeah, looking back at this code - it looks a bit over-engineered and needs some drastic simplications.